src/applications/caregivers/components/AdditionalInfo/index.jsx

```diff
@@ -1,16 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
 import { links } from 'applications/caregivers/definitions/content';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
 
 export const VetInfo = () => (
-  <p>
-    Complete the following information about the Veteran or Service member. You
-    aren’t required to fill in all fields, but we can review your application
-    faster if you provide more information.
-  </p>
+  <p>Please complete all of the following information</p>
 );
 
+export const LearnMoreFamilyCaregiver = () => {
+  const [isModalVisible, toggleModal] = useState(false);
+
+  return (
+    <>
+      <Modal
+        title="Primary Family Caregiver"
+        id="primary-family-caregiver-info"
+        cssClass="caregiver-modal"
```

Comment:
I really wish we could switch this to className or something similar - not related to your work but this is an odd prop name

Reply:
Yeah I totally agree, but should isolate the change in another PR, but I feel ya fo sho

Reply:
There is a lot of repeated content here. Would you find a way to make it more DRY?